database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningAdditionalDataIntegrationTests.scala

```diff
@@ -88,19 +88,21 @@ class GetPartitioningAdditionalDataIntegrationTests extends DBTestSuite{
     )
 
     function(fncGetPartitioningAdditionalData)
-      .setParam("i_partitioning", partitioning1)
+      .setParam("i_partitioning_id", fkPartitioning1)
```

Comment:
You know, that you don't have to name the params, you can put them there positionally? Especially in one param functions, it's easier

Reply:
Yep, I know, but I find it nicer - more readable if it's named, especially if there are more functions; but for 1-2 params I think you are right, can be ommited